ese-varo/growing
# feature/last-day-reminder

## Explain why this change is being made.
As a user with a habit close to finish, I would like to receive a notification to remind me to record a summary about my habit record so i can have a retrospective about the habit

## Explain how this is accomplished.
- Given the user receives an email with the habit information and a link to that habit page when the habit's end date is the current day
- Then the user can click on that link and go to that habit page and then the user can
  - Mark that habit as created or not created
  -  Add final notes for the habit

## How do you manually test this?
Everyday the app will check for all the habits that is in the last day and is not check as achieved, so you will receive an email with the data and link to the habit

## Screenshots
![image](https://user-images.githubusercontent.com/32284972/115473790-9265b600-a1f9-11eb-881f-319040394866.png)
![image](https://user-images.githubusercontent.com/32284972/115473805-9a255a80-a1f9-11eb-9b40-36da4fc3f3a6.png)
![image](https://user-images.githubusercontent.com/32284972/115473827-a5788600-a1f9-11eb-9eb8-cab6b127b8ca.png)
![image](https://user-images.githubusercontent.com/32284972/115473834-aa3d3a00-a1f9-11eb-87bf-881b2f3e46e7.png)
![image](https://user-images.githubusercontent.com/32284972/115473852-b2957500-a1f9-11eb-80df-56f35e658617.png)
